mailpoet/lib/Automation/Engine/Storage/WorkflowStorage.php

```diff
@@ -85,4 +148,24 @@ public function getActiveWorkflowsByTrigger(Trigger $trigger): array {
       return Workflow::fromArray($workflowData);
     }, (array)$data);
   }
+
+  private function workflowHeaderData(Workflow $workflow): array {
```

Comment:
To satisfy our naming conventions, I guess it needs a verb, such as `getWorkflowHeaderData`.

Reply:
Addressed in [f8785cd](https://github.com/mailpoet/mailpoet/pull/4289/commits/f8785cd3ad72147ab76f556977f32919d2619ea2)